cmd/cayley/command/http.go

```diff
@@ -46,7 +46,6 @@ func NewHttpCmd() *cobra.Command {
 	cmd.Flags().String("host", "127.0.0.1:64210", "host:port to listen on")
 	cmd.Flags().Bool("init", false, "initialize the database before using it")
 	cmd.Flags().DurationP("timeout", "t", 30*time.Second, "elapsed time until an individual query times out")
-	cmd.Flags().StringVar(&chttp.AssetsPath, "assets", "", "explicit path to the HTTP assets")
```

Comment:
Can we preserve this option and set it to Packr somehow? Since paths given to Packr boxes are `../../xxx`, resolving local UI files won't work if they are in the same dir.

Reply:
When will they be in the same dir? Assuming the assets are provided inside the binary.

Reply:
It's probably a good idea to do it in Docker, for example. The point is that we can override the embedded UI for local development, or use UI with a build that has none embedded into it.

Reply:
I think that if you want to use custom UI it’s legit it will be served by a different process. For instance: Cayley Web has it’s own dev server

Reply:
Fair enough. Then lets set this flag to a new variable (`.StringVar(&assetsPath, ...`) and fail the CLI with an error if the value of the flag is not empty. This will at least make sure that someone who have used the flag to override the UI gets an error message explaining that the flag is deprecated now.

Reply:
Done